<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoyStickDlg</class>
 <widget class="QWidget" name="JoyStickDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip{
	color : black;
}
QWidget{
	background : #333333;
}
QGroupBox{
	color : white;
	margin-top : 2ex;
	border-radius : 5px;
	padding : 8px;
	padding-top : 13px;
	 border-style: outset;
 	 border-width: 2px;
	 border-color: #1f1f1f;
}
QMenuBar#menuBar{

	background : grey;
	color : white;
	 border-bottom-style: outset;
 	 border-bottom-width: 1px;
	 border-bottom-color: #222222;
}
QMenu{
	background : grey;
	color : #1f1f1f;
}
QMenu::item:selected{
	background : #c9c9c9;
}
QStatusBar#statusBar{
	background : grey;
	color : white;
	border-top-style: outset;
 	border-top-width: 1px;
	border-top-color: #222222;
}
QPushButton{
	color : white;
	background : grey;
	padding : 5px;
	border-radius : 3px;
}

QPushButton:hover{
	background : #c9c9c9;
}
QPushButton:pressed{
	background : #474747;
}
QPushButton:disabled{
	color : #474747;
}
QLineEdit{
	color : white;
	background : #111111;
	padding : 3px;
	border-radius : 3px;
	border-color : white;
	border-width : 1px;
}
QComboBox{
	color : white;
	background : #111111;
	padding : 3px;
	min-width : 40px;
	border-radius : 3px;

}
QComboBox::drop-down{
	
}
QComboBox QAbstractItemView{
	background-color : black;
	color : white;
}
QSlider::handle{
	background : grey;
}
QSlider::handle:hover{
	background : #c9c9c9;
}
QSlider::handle:pressed{
	background : #474747;
}
QLabel{
	color : white;
}
QRadioButton{
	color : white;
}
QCheckBox{
	color : white;
}
QRadioButton::indicator{
	 border-style: outset;
 	 border-width: 2px;
	 border-color: #1f1f1f;
	background : white;
	border-radius : 6px;
}
QRadioButton::indicator:checked{
	 border-style: outset;
 	 border-width: 2px;
	 border-color: white;
	background : #1f1f1f;
	border-radius : 6px;
}
QCheckBox::indicator{
	 border-style: outset;
 	 border-width: 2px;
	 border-color: #1f1f1f;
	background : white;
}
QCheckBox::indicator:checked{
	 border-style: outset;
 	 border-width: 2px;
	 border-color: white;
	background : #1f1f1f;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select JoyStick:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="joystickCombo"/>
   </item>
   <item>
    <widget class="QCheckBox" name="useJoyStickCheck">
     <property name="text">
      <string>Use JoyStick</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useHotKeyCheck">
     <property name="text">
      <string>Use HotKeys</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
